tools/stern/src/Stern/Intra.hs

```diff
@@ -454,18 +455,18 @@ getTeamBillingInfo :: TeamId -> Handler (Maybe TeamBillingInfo)
 getTeamBillingInfo tid = do
   info $ msg "Getting team billing info"
   i <- view ibis
-  r <-
+  resp <-
     catchRpcErrors $
       rpc'
         "ibis"
         i
         ( method GET
             . Bilge.paths ["i", "team", toByteString' tid, "billing"]
         )
-  case Bilge.statusCode r of
-    200 -> Just <$> parseResponse (mkError status502 "bad-upstream") r
+  case Bilge.statusCode resp of
+    200 -> Just <$> parseResponse (mkError status502 "bad-upstream") resp
     404 -> pure Nothing
-    _ -> throwE (mkError status502 "bad-upstream" "bad response")
+    _ -> throwE (mkError status502 "bad-upstream" (errorMessage resp))
```

Comment:
i thought the idea was to re-throw the upstream error message?  isn't that more helpful than wrapping that in a 5xx?

Reply:
I thought we just didn't want to lose the meta information from the error response, but still re-throw upstream issues as 502s.